src/modules/core/components/Fields/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useField } from 'formik';
+import { PopperProps } from 'react-popper';
+
+import InputLabel from '~core/Fields/InputLabel';
+import Icon from '~core/Icon';
+import { Tooltip } from '~core/Popover';
+
+import styles from './Toggle.css';
+
+const displayName = 'Toggle';
+
+interface Props {
```

Comment:
This needs `appearance` themes.

Off the top of my head:
- primary (switch is green)
- danger (switch is pink)
- minimal (doesn't have an outline, just a grey backround like the progress bar does)

But these can be all added in time, when they are needed. 

For now, please just implement the first two: primary and danger